src/DynamoCoreWpf/Extensions/ViewLoadedParams.cs

```diff
@@ -22,12 +24,16 @@ public class ViewLoadedParams : ReadyParams
         private readonly DynamoViewModel dynamoViewModel;
         public readonly Menu dynamoMenu;
 
+        public Watch3DView BackgroundPreView { get { return dynamoView.BackgroundPreview; } }
```

Comment:
Exposing `Watch3DView` as a property in `ViewLoadedParams` class so that it can be passed to `DynamoManipulationExtension` as a manipulator context.